onyo/lib/inventory.py

```diff
@@ -857,12 +857,17 @@ def get_items(self,
                 # report the error, and proceed
                 ui.error(e)
 
-    def get_asset_from_template(self,
-                                template: Path | str | None) -> Item:
-        r"""Get a template as an Item."""
-
-        # TODO: Possibly join with get_asset_content()
-        return Item(self.repo.get_template(template))  # , repo=self.repo ?? Probably not. Template is not yet bound.
+    def get_templates(self,
+                      template: Path | None,
+                      recursive: bool = False) -> Generator[Item, None, None]:
+        r"""Get templates as Items."""
```

Comment:
I don't want to get you bogged down in docstrings. But it is preferable if the parameters are also documented.

Not a blocker.